sparql/allRevisions.q

```diff
@@ -0,0 +1,18 @@
+select *
+where{
+    graph ?record {
+        ?revision rev:describes ex:DocumentId .
+        optional {
+            ?revision rev:isNewRevisionOf ?oldRevision .
+        }
+        ?revision rev:containsRecord ?containedRecord .
+    }
+    filter not exists{
+        graph ?newerRecord {
+            ?newerRecord rec:replaces ?record .
+        }
+    }
+    graph ?containedRecord {
```

Comment:
I assume the query is meant to show content of revisions, but it returns all data in all revisions (multiple times), which makes it time out.